test/integration/dynamic_validation_integration_test.cc

```diff
@@ -96,7 +96,7 @@ INSTANTIATE_TEST_SUITE_P(
 // Protocol options in CDS with unknown fields are rejected if and only if strict.
 TEST_P(DynamicValidationIntegrationTest, CdsProtocolOptionsRejected) {
   api_filesystem_config_ = {
-      "test/config/integration/server_xds.bootstrap.yaml",
+      "test/config/integration/server_xds.bootstrap.yml",
```

Comment:
Optional nit: technically an orthogonal test for this would be cleaner, but not that big a deal.